x-pack/plugins/secops/public/components/app_settings/app_settings_popover.test.tsx

```diff
@@ -4,14 +4,23 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { mount } from 'enzyme';
+import { mount, shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
```

Comment:
Looks like there's a `shallowToJson` that the [docs](https://github.com/adriantoine/enzyme-to-json/blob/df30a5ace977c97fb42a6164a9f47af618ccf368/docs/helper-functions.md#use-individual-helper-functions) recommend you use. Appears to be just for semantics though, as [shallowToJson](https://github.com/adriantoine/enzyme-to-json/blob/df30a5ace977c97fb42a6164a9f47af618ccf368/src/shallow.js#L58) matches [mountToJson](https://github.com/adriantoine/enzyme-to-json/blob/df30a5ace977c97fb42a6164a9f47af618ccf368/src/mount.js#L67) and `toJson` just passes to the appropriately named function based on wrapper type. No changes necessary, but we could use this going forward.

Reply:
Yea, seems like such a `no-op` and not much semantic between it and `toJson` though. I guess people can use that one later but `toJson` works with both shallow and mount and is less characters too?